tools-testing/cross-browser-testing/javascript/fetch-broken/script.js

```diff
@@ -0,0 +1,52 @@
+const header = document.querySelector("header");
+const section = document.querySelector("section");
+
+const requestURL =
+  "https://mdn.github.io/learning-area/javascript/oojs/json/superheroes.json";
+
+const response = fetch(requestURL);
+populateHeader(response);
+showHeroes(response);
+
+function populateHeader(jsonObj) {
+  const myH1 = document.createElement("h1");
+  myH1.textContent = jsonObj["squadName"];
```

Comment:
NIT. I don't like names starting with _my_; in production they are almost never the right name to chose, so it is better not to have learner use it.

```suggestion
  const h1 = document.createElement("h1");
  h1.textContent = jsonObj["squadName"];
```